tella-locking-ui/src/main/java/com/hzontal/tella_locking_ui/common/util/DivviupUtils.kt

```diff
@@ -34,7 +43,302 @@ object DivviupUtils {
                 Timber.e(e, "Divviup sending runUnlockEvent failed")
             }
         }
+    }
+
+    /**
+     * This should be called only once when user enables sending metrics
+     * Which means that the installation is done and the sending of analytics is approved
+     **/
+    fun runInstallEvent(context: Context) {
+        // If sending analytics is not accepted or install metric is sent, just return
+        if (!CommonPreferences.hasAcceptedAnalytics() || CommonPreferences.isInstallMetricSent() || isDebugBuild(context)) return
+        Executors.newSingleThreadExecutor().execute {
+            try {
+                val taskId = TaskId.parse(context.getString(R.string.divviup_count_installs_id))
+                val leaderEndpoint: URI =
+                    URI.create(context.getString(R.string.divviup_leader))
+                val helperEndpoint: URI =
+                    URI.create(context.getString(R.string.divviup_helper))
+                val timePrecisionSeconds: Long =
+                    context.resources.getInteger(R.integer.divviup_count_unlocks_timePrecisionSeconds)
+                        .toLong()
+                val client = Client.createPrio3Count(
+                    context, leaderEndpoint, helperEndpoint, taskId, timePrecisionSeconds
+                )
```

Comment:
we can also consider injecting the client using dagger and intialize it only once, we can provide leaderEndpoint and helperEndpoint easily too as we have static strings here .. 

 ```
 @Provides
    fun provideClient(
        context: Context,
        @Named("leaderEndpoint") leaderEndpoint: URI,
        @Named("helperEndpoint") helperEndpoint: URI,
        taskId: TaskId,
        timePrecisionSeconds: Long
    ): Client {
        return Client.createPrio3Count(
            context, leaderEndpoint, helperEndpoint, taskId, timePrecisionSeconds
        )
    }
```